docker-compose.yml

```
@@ -129,9 +129,9 @@ services:
     image: ${NEO4J_DOCKER_IMAGE:-neo4j:5.19.0-community}
     restart: unless-stopped
     environment:
-      - "NEO4J_AUTH=neo4j/admin"
-      - "NEO4J_dbms_security_procedures_unrestricted=apoc.*"
-      - "NEO4J_dbms_security_auth__minimum__password__length=4"
+      NEO4J_AUTH: neo4j/${INFRAHUB_DB_PASSWORD}
```

```suggestion
      NEO4J_AUTH: neo4j/${INFRAHUB_DB_PASSWORD:-admin}
```

You should use the same syntax as above in the x-infrahub-config block, to keep default values when the env var is not set.

---

For some reason we can't change NEO4J login ... https://neo4j.com/docs/operations-manual/current/docker/introduction/#docker-auth
(I'll set default password, shouldn't it be 'infrahub' tho?)

---

I adjusted the part with password, for the login I don't know .... So far it sounds pretty much like a trap as if the user changes the var `INFRAHUB_DB_USERNAME` it just won't work ... Should we add a comment like "If you use neo4j please don't change that" or make sure that whrn using neo4j this var doesn't exist ...

---

Right I didn't know about that thanks for the pointer. Then I think we can leave it as-is...